docs/appendices/release-notes/5.6.3.rst

```diff
@@ -47,4 +47,5 @@ See the :ref:`version_5.6.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed ``NullPointerException`` thrown when joining tables with ``USING``
+  clause which contained columns not known to one side of the tables.
```

Comment:
phrasing suggestion:
```suggestion
  clause which contains columns not existing in either or both tables.
```
At least `not known` -> `not existing` reads better imho.

Reply:
Thank you! updated.